### DISCUSSION TITLE
When rendering, some level in some frames fails

### DISCUSSION THREAD

**Discussion by WanderStudios** [opentoonz/opentoonz#6771] [Troubleshooting]
### Issue Summary
Render PNG seq 8 bit with alpha.
3840x2160
Ramdomly, when rendering at some frames there is not all levels selected.
It occurs in multiple scenes, and multiples levels.

### Screenshot or Video Reference
![toonfail](https://user-images.githubusercontent.com/61146444/74651633-89f09600-5184-11ea-8937-fbb1f15d376f.jpg)

### Steps to Reproduce
When press render, it occurs randomly

### Expected Results
All selected (yellow) levels visible on render files

### Actual Results
Some level in some frames fails

### System Information
* **OpenToonz Version**: 1.4 (built Feb 2 2020 00:30:54)
* **Operating System**: Windows 10 Pro 64-bit
* **CPU**: Intel Xeon E5640 @ 2.67GHz (2 cpu x 4 cores)
* **Memory**: 96 GB
* **Graphics Card**: Quadro M5000
* **Graphics Tablet**: Wacom Cintiq 22HD

**Comment by RodneyBaker** [comment]
There is a known issue with 8bit PNGs.
Perhaps this relates to what you are experiencing.

REF:  #242

**Comment by WanderStudios** [comment]
The issue isn't load a PNG. The issue is render to PNG.

**Comment by artisteacher** [comment]
When you play through in the viewer, does everything display as expected? Do the drawings for each level all look complete in the level strip?

**Comment by WanderStudios** [comment]
Yes, when I play throught it is ok.

**Comment by WanderStudios** [comment]
I had the same issue in v1.3

**Comment by artisteacher** [comment]
Is it possible that some of the drawings unintentionally ended up on a different level (that has render visibility turned off)?

**Comment by WanderStudios** [comment]
No, you can see it in the screenshot.
If I press render, the frames that "disappear" are random. Sometimes render all frames ok, sometimes no.

**Comment by artisteacher** [comment]
Does the level type seem to matter? Does rendering to a different format make a difference?

**Comment by WanderStudios** [comment]
Hi, I tried other formats and the same result.
I've discovered something: if I play all the frames before rendering it works fine. But if I render just when opening the project is when it does not work well. If I press on one of the frames that do not show well, the viewfinder does not show well either and I have to go one frame before and then return to the same frame so that it loads the draw well. Will it be something from the cache?
